extensions/agroal/runtime/pom.xml

```diff
@@ -34,6 +34,10 @@
             <artifactId>svm</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.opentracing.contrib</groupId>
+            <artifactId>opentracing-jdbc</artifactId>
+        </dependency>
```

Comment:
This is a problem, we cannot have a hard dependency on OpenTracing here.

Reply:
Should I add it in a different scope ?